static/docs/commands-reference/config.md

```diff
@@ -187,7 +187,7 @@ Add an S3 remote and set it as the project default:
 > to create your bucket.
 
 ```dvc
-$ dvc remote add myremote s3://bucket/path
```

Comment:
why do we change this? even though it's a key in S3 it's easier to understand as a path I believe

Reply:
It's just a more correct term in the context of S3 but I can revert if you prefer.

Justification: Being used to S3 I always think about "keys" not "paths". These docs for ex. only talk about keys (and don't even mention paths): https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html and https://docs.aws.amazon.com/AmazonS3/latest/dev/Introduction.html. The term "path" is used in the S3 documentation in the context of bucket access URLs, which is not what we're referring to in the example above, e.g. https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html

Lmk )

Reply:
I still think it's better to keep path here. We don't talk about a specific key. It's more like a prefix for all object we will put later. So, it's confusing to see a key here. Also, my intuition is that people less familiar with S3 internals.

Reply:
Sure, reverted.